Starforge/Util/MiscHelper.cs

```diff
@@ -85,7 +86,7 @@ public static class MiscHelper {
         }
 
         public static int RandInt(int seed, int max) {
-            return RandBytes[seed] % max;
+            return RandBytes[seed] % (max % ushort.MaxValue);
```

Comment:
I think you mixed something up here, you wanna do
```suggestion
            return RandBytes[seed % ushort.MaxValue] % max;
```
Since RandBytes only contains bytes, i.e. values from 0 to 255 that `% (max % ushort.MaxValue)` would only have questionable effects in rare cases.